desktop/cmp/rest/RestGridModel.js

```diff
@@ -102,7 +109,8 @@ export class RestGridModel {
      * @param {Object[]|RecordAction[]} [toolbarActions] - actions to display in the toolbar. Defaults to add, edit, delete.
      * @param {Object[]|RecordAction[]} [menuActions] - actions to display in the grid context menu. Defaults to add, edit, delete.
      * @param {Object[]|RecordAction[]} [formActions] - actions to display in the form toolbar. Defaults to delete.
-     * @param {Object} [actionWarning] - map of action (e.g. 'add'/'edit'/'delete') to string.  See default prop.
+     * @param {Object} [actionWarning] - map of action (e.g. 'add'/'edit'/'del') to string or a fn to create one. See default prop.
+     *      Function will be passed selected record(s) to be acted upon.
```

Comment:
probably simplest to always pass an *array* of records for *all* actions, and specify that you are doing so here.  This will mean the developer doesn't have to guess based on the action whether it receives one or all records.  Edit, for axample, could be multi-record enabled at some point in the future.